### PR TITLE
add more info about testing and where tests live

### PR DESCRIPTION
**Pull request by diasdavid**
Avoid creating confusion - https://github.com/ipfs/js-ipfs-api/pull/322

@RichardLitt wanna review?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +100,29 @@ $ ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin "[\"http://exam
...
+
+`js-ipfs-api` follows the spec defined by [`interface-ipfs-core`](https://github.com/ipfs/interface-ipfs-core), which governs the interface to expect from IPFS implementations. This interface is an current active endeavour, expect it to be complete in the following weeks (August 2016). You can use it today to consult the methods available.
+
+### Extra API methods
+
+Adding to the methods defined by [`interface-ipfs-core`](https://github.com/ipfs/interface-ipfs-core), `js-ipfs-api` exposes a set of extra utility methods.
+
+A complete documentation for these methods is coming with: https://github.com/ipfs/js-ipfs-api/pull/305
+
+#### `ipfs.util.addURL`
```
Comment (dignifiedquire) [inline README.md]: These methods don't exist yet

Comment (diasdavid) [inline README.md]: fair. Avoiding referencing them then

**Comment by dignifiedquire** [comment]
LGTM

**Comment by diasdavid** [comment]
Thank you :)

**Comment by RichardLitt** [comment]
@diasdavid Fixed some things. :)

**Pull request merged**